[2025-08-18T02:53:33.092+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-18T02:53:33.219+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.run_dbt_transformations manual__2025-08-18T01:29:52.036937+00:00 [queued]>
[2025-08-18T02:53:33.224+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.run_dbt_transformations manual__2025-08-18T01:29:52.036937+00:00 [queued]>
[2025-08-18T02:53:33.225+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-08-18T02:53:33.232+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_dbt_transformations> on 2025-08-18 01:29:52.036937+00:00
[2025-08-18T02:53:33.239+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'run_dbt_transformations', 'manual__2025-08-18T01:29:52.036937+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/weather_dags.py', '--cfg-path', '/tmp/tmp_z5f62rx']
[2025-08-18T02:53:33.241+0000] {standard_task_runner.py:91} INFO - Job 70: Subtask run_dbt_transformations
[2025-08-18T02:53:33.241+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=379) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-18T02:53:33.241+0000] {standard_task_runner.py:63} INFO - Started process 381 to run task
[2025-08-18T02:53:33.284+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.run_dbt_transformations manual__2025-08-18T01:29:52.036937+00:00 [running]> on host b10b13b5f146
[2025-08-18T02:53:33.358+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Hendi Irwanto' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_transformations' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T01:29:52.036937+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T01:29:52.036937+00:00'
[2025-08-18T02:53:33.358+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-18T02:53:33.367+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T02:53:33.368+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --project-dir /opt/data-engineer/weather-project/dbt/weather_dbt --profiles-dir /opt/data-engineer/weather-project/dbt/weather_dbt/.dbt']
[2025-08-18T02:53:33.375+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T02:53:34.792+0000] {subprocess.py:93} INFO - [0m02:53:34  Running with dbt=1.10.7
[2025-08-18T02:53:35.074+0000] {subprocess.py:93} INFO - [0m02:53:35  Registered adapter: postgres=1.9.0
[2025-08-18T02:53:35.281+0000] {subprocess.py:93} INFO - [0m02:53:35  Unable to do partial parsing because profile has changed
[2025-08-18T02:53:36.768+0000] {subprocess.py:93} INFO - [0m02:53:36  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-08-18T02:53:36.769+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-08-18T02:53:36.769+0000] {subprocess.py:93} INFO - - models.dbt_date
[2025-08-18T02:53:36.867+0000] {subprocess.py:93} INFO - [0m02:53:36  Found 5 models, 2 sources, 576 macros
[2025-08-18T02:53:36.869+0000] {subprocess.py:93} INFO - [0m02:53:36
[2025-08-18T02:53:36.869+0000] {subprocess.py:93} INFO - [0m02:53:36  Concurrency: 1 threads (target='dev')
[2025-08-18T02:53:36.870+0000] {subprocess.py:93} INFO - [0m02:53:36
[2025-08-18T02:53:37.015+0000] {subprocess.py:93} INFO - [0m02:53:37  1 of 5 START sql table model dimension.dim_dates ............................... [RUN]
[2025-08-18T02:53:37.154+0000] {subprocess.py:93} INFO - [0m02:53:37  1 of 5 OK created sql table model dimension.dim_dates .......................... [[32mSELECT 1837[0m in 0.14s]
[2025-08-18T02:53:37.155+0000] {subprocess.py:93} INFO - [0m02:53:37  2 of 5 START sql table model dimension.dim_locations ........................... [RUN]
[2025-08-18T02:53:37.189+0000] {subprocess.py:93} INFO - [0m02:53:37  2 of 5 OK created sql table model dimension.dim_locations ...................... [[32mSELECT 100[0m in 0.03s]
[2025-08-18T02:53:37.190+0000] {subprocess.py:93} INFO - [0m02:53:37  3 of 5 START sql view model staging.stg_weather_data ........................... [RUN]
[2025-08-18T02:53:37.287+0000] {subprocess.py:93} INFO - [0m02:53:37  3 of 5 OK created sql view model staging.stg_weather_data ...................... [[32mCREATE VIEW[0m in 0.10s]
[2025-08-18T02:53:37.289+0000] {subprocess.py:93} INFO - [0m02:53:37  4 of 5 START sql table model marts.mart_avg_temp_by_day_of_week ................ [RUN]
[2025-08-18T02:53:37.326+0000] {subprocess.py:93} INFO - [0m02:53:37  4 of 5 OK created sql table model marts.mart_avg_temp_by_day_of_week ........... [[32mSELECT 700[0m in 0.04s]
[2025-08-18T02:53:37.327+0000] {subprocess.py:93} INFO - [0m02:53:37  5 of 5 START sql table model marts.mart_daily_weather .......................... [RUN]
[2025-08-18T02:53:37.370+0000] {subprocess.py:93} INFO - [0m02:53:37  5 of 5 OK created sql table model marts.mart_daily_weather ..................... [[32mSELECT 1700[0m in 0.04s]
[2025-08-18T02:53:37.382+0000] {subprocess.py:93} INFO - [0m02:53:37
[2025-08-18T02:53:37.382+0000] {subprocess.py:93} INFO - [0m02:53:37  Finished running 4 table models, 1 view model in 0 hours 0 minutes and 0.51 seconds (0.51s).
[2025-08-18T02:53:37.416+0000] {subprocess.py:93} INFO - [0m02:53:37
[2025-08-18T02:53:37.417+0000] {subprocess.py:93} INFO - [0m02:53:37  [32mCompleted successfully[0m
[2025-08-18T02:53:37.417+0000] {subprocess.py:93} INFO - [0m02:53:37
[2025-08-18T02:53:37.418+0000] {subprocess.py:93} INFO - [0m02:53:37  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 NO-OP=0 TOTAL=5
[2025-08-18T02:53:38.944+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-18T02:53:38.944+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-18T02:53:38.972+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=run_dbt_transformations, run_id=manual__2025-08-18T01:29:52.036937+00:00, execution_date=20250818T012952, start_date=20250818T025333, end_date=20250818T025338
[2025-08-18T02:53:38.992+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-18T02:53:39.006+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-18T02:53:39.007+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
